.github/matrix.json

```diff
@@ -1,5 +1,5 @@
 {
   "postgresql": ["12"],
-  "ruby": ["2.7"],
+  "ruby": ["2.7", "3.0"],
```

Comment:
Can we separate this out to its own commit? I'd like to merge all the preparation work before we pull the trigger.